src/ensembl/src/content/app/help/components/help-menu/HelpMenu.tsx

```diff
@@ -58,32 +62,41 @@ const HelpMenu = (props: Props) => {
       clickedMenuRef.current = menuIndex;
       nextValue = items;
     } else {
-      // this means a repeated click on the same menu iteem
+      // this means a repeated click on the same menu item
       clickedMenuRef.current = null;
     }
-
     setSubmenuItems(nextValue);
   };
 
+  const handleMenuItemClick = (menuName: string) => {
+    const shouldTrack = clickedMenuRef.current || menuName === 'Overview';
+    shouldTrack && trackMegaNavOpen(menuName);
+  };
+
   const closeMegaMenu = () => {
     setSubmenuItems(null);
     clickedMenuRef.current = null;
   };
 
   const topLevelItems = props.menu.items.map((item, index) => {
     const className = classNames(styles.topMenuItem);
-    const commonProps = {
-      key: index,
-      className
-    };
-    return item.type === 'collection' ? (
-      <span {...commonProps} onClick={() => toggleMegaMenu(item.items, index)}>
-        {item.name}
-      </span>
-    ) : (
-      <HelpMenuLink {...commonProps} to={item.url} onClick={closeMegaMenu}>
-        {item.name}
-      </HelpMenuLink>
+
+    return (
+      <div
+        key={index}
+        className={className}
+        onClick={() => handleMenuItemClick(item.name)}
+      >
```

Comment:
Mmm, I don't like that in order to add a common onclick behaviour the elements are wrapped in another DOM element. Looks like it would be sufficient simply to modify the click handlers.